bin/hokusai

```diff
@@ -173,17 +179,29 @@ def stack(action, staging, production, verbose):
 @click.option('-v', '--verbose', type=click.BOOL, is_flag=True, help='Verbose output')
 def deploy(tag, staging, production, verbose):
   """
-  Deploy an image tag and update the tag (staging/production) to reference the same image
+  Update the project's deployment(s) to reference a named image tag and update the tag (staging/production) to reference the same image
   """
   hokusai.lib.common.set_output(verbose)
   context = select_context(staging, production)
   hokusai.deploy(context, tag)
 
 @cli.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--staging', type=click.BOOL, is_flag=True, help='Target staging')
+@click.option('--production', type=click.BOOL, is_flag=True, help='Target production')
+@click.option('-v', '--verbose', type=click.BOOL, is_flag=True, help='Verbose output')
+def refresh(staging, production, verbose):
```

Comment:
@cavvia so updating the `deploymentTimestamp` is a hack used to trigger a deployment rollout so that it creates a new replicaSet (i.e. spawns new pods) even when the image, or other configuration has not changed.  It should be handled under the hood.